libsolidity/formal/SolverInterface.h

```diff
@@ -202,6 +227,22 @@ class Expression
 		);
 	}
 
+	static Expression const_array(Expression _sort, Expression _value)
+	{
+		solAssert(_sort.sort->kind == Kind::Sort, "");
+		std::shared_ptr<SortSort> sortSort = std::dynamic_pointer_cast<SortSort>(_sort.sort);
```

Comment:
Perhaps use `auto` here?

Reply:
Why? ;)

Reply:
Just because the type should be clear here and it would save some characters ;) But it's fine.